guides/v2.1/cloud/project/project-integrate-blackfire.md

```diff
@@ -254,7 +262,12 @@ When you configure at least one way of triggering builds with Blackfire, you can
 
 ## Blackfire support
 
-If you continue to experience problems, you can contact Blackfire support and provide output from the following:
+If you continue to experience problems, you can contact Blackfire support and provide output.
```

Comment:
Consider:  If you continue to experience problems, save your log files and send them Blackfire support for help.  (or something like that -- sounds a bit awkward now.)

Consider including link to the Blackfire support to make it easy for people.
https://support.blackfire.io/questions-about-blackfire/contact-us/contact-us